scripts/system_tests_utils.ts

```diff
@@ -250,8 +250,9 @@ export const createNewAccount = async (config?: {
 			const url = getSystemTestProviderUrl();
 			const web3 = new Web3(url);
 			web3.registerPlugin(new HardhatPlugin());
+			// eslint-disable-next-line
 			await web3.hardhat.impersonateAccount(acc.address);
-			// await impersonateAccount(acc.address);
+			// eslint-disable-next-line
```

Comment:
seems scripts are not eslinted till you modify them and then all previous issues come up (on git commit)